projects/ffmpeg/build.sh

```diff
@@ -22,16 +22,17 @@ if [[ $CONFIG = "libfuzzer" ]]
 then
       export CC="clang"
       export CXX="clang++"
-      export LDFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+      export FFMPEG_CFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+      export LDFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,float-divide-by-zero"
```

Comment:
Is it possible to instrument dependencies for UbSAN too?

Reply:
added ubsan for C deps